composer.json

```diff
@@ -1,25 +1,31 @@
 {
-    "name": "xenos/musicbrainz",
+    "name": "papposilene/musicbrainz",
     "type": "library",
     "description": "A PHP library for accessing the MusicBrainz API",
     "keywords": ["musicbrainz", "library", "php", "api"],
-    "homepage": "https://github.com/XenosEleatikos/MusicBrainz",
+    "homepage": "https://github.com/papposilene/MusicBrainz",
     "license": "MIT",
     "authors": [
         {
             "name": "Christoph Hochholzer",
             "email": "christoph@hochholzer.info",
             "homepage": "https://github.com/XenosEleatikos",
             "role": "Developer"
+        },
+        {
+            "name": "Philippe-Alexandre Pierre",
+            "email": "dev@psln.nl",
+            "homepage": "https://github.com/papposilene",
+            "role": "Developer"
         }
     ],
     "require": {
         "php": ">=7.1.0",
+        "guzzlehttp/guzzle": "^6",
```

Comment:
I've had it only in dev requirements, because this library uses adapter pattern and implementation is againts AbstractHttpAdapter. I would prefer to let it as it is.